components/MainNav.jsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
 import styled from 'styled-components';
+import { useSpring, animated } from 'react-spring'
 import {
     BrowserRouter as Router,
     Link
 } from 'react-router-dom';
 
-const MainNav = ({ setIsNav, setIsSpeechBoxActive }) => {
+const MainNav = ({ setIsNav, isNav, setIsSpeechBoxActive }) => {
+    const animationProps = useSpring({ 
+        to: { scaleY: 1 }, 
+        from: { scaleY: 0 },
+    })
+
+    const AnimatedNavPane = animated(NavPane)
```

Comment:
Ok, so this library is a wrapper around our components that allows us to pass animation information as a prop. That prop is an object wrapped around a hook that I'm guessing makes it digestible for our newly wrapped component.

Here's a great opportunity to name this something more than `animationProps`. What are we animating? Name your animation props that.